File: backend/tests/fixtures/parsers/scala/hello.scala
package com.example

import scala.collection.mutable

class Greeter extends Base {
  def greet(name: String): Unit = {
    val formatted = formatName(name)
    println(formatted)
  }

  def formatName(name: String): String = name.toUpperCase
}

object Main {
  def run(): Unit = {
    new Greeter().greet("world")
  }
}

File: backend/tests/fixtures/parsers/elixir/hello.ex
defmodule Greeter do
  def greet(name) do
    format_name(name)
  end

  def format_name(n) do
    String.upcase(n)
  end
end

defmodule Main do
  def run do
    Greeter.greet("world")
  end
end
